### ISSUE TITLE
some articles are not summarized

### ISSUE THREAD

**Issue by thiswillbeyourgithub** [polyrabbit/hacker-news-digest#21]
For example : https://www.smithsonianmag.com/history/the-photographer-who-forced-the-us-to-confront-its-child-labor-problem-180982355/

The summary is extremely long and unformatted. I've seen this several times so I think there is a bug in your code for some websites of input file type?

Thanks for the website btw

**Comment by polyrabbit** [comment]
Hi, thanks for the feedback. This article is already summarized by OpenAI, as denoted by the model icon. But the returned summary is longer than 400 characters, so I have to truncate and append an ellipsis to the end.

Quote for the official doc:
> Note however that instructing the model to generate a specific number of words does not work with high precision.

<img width="898" alt="image" src="https://github.com/polyrabbit/hacker-news-digest/assets/2657334/625e6524-672f-46a9-a564-8fc50bc527e9">

**Comment by thiswillbeyourgithub** [comment]
Hi, thanks for the quick answer but I'm confident there is a bug.

I am using your website via RSS and when I click on most links I see that the content is indeed a summary, but for the article I linked the summary is pages and pages long, a wall of text.

Here's a picture : 

Normal article summary: 
![image](https://github.com/polyrabbit/hacker-news-digest/assets/26625900/d7e08930-2341-4f73-8bf0-7a67bcc04273)

Suspicious article: 
![image](https://github.com/polyrabbit/hacker-news-digest/assets/26625900/a2d1fce1-cb2e-4f79-9f3b-f362dc1594f1)
![image](https://github.com/polyrabbit/hacker-news-digest/assets/26625900/fd86ded8-e57d-4cbf-9ca8-82fe96e3fd10)


I only supplied the first and last page but the whole wall of text is about 5 times the content of a single screenshot.
